docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
+
+Description
+-----------
+
+The SimpleCheck gives you basic information about your datasets and detects the most obvious of datashifts.
+
+Some shifts result in the change of basic metrics, that are commonly used for the analysis and exploration of datasets.
+Those metrics and statistics are calculated on both datasets and compared afterwards. HIER NOCH AMAZON DEEQU ERWÄHNEN,
+WIRD NACH UMBENENNUNG DES CHECKS GEMACHT
+
+The basic check should be used in most analyses because its metrics are scientifically grounded and commmonly used.
```

Comment:
I don't like this sentence - I don't think that 
```suggestion
:ref:`simple` should be used in most analyses because its metrics are scientifically grounded and commmonly used.
```

Reply:
Alright, I weakend it a little bit. 
"The basic check should be used in most analyses, its metrics are commonly used to describe datasets."